client/src/components/pages/alarm.tsx

```diff
@@ -18,26 +18,7 @@ type AlarmType = {
 
```

Comment:
add back:

```typescript
useEffect(() => {
        document.title = "Sleep Buddies - Alarms";
    }, []);
```

Reply:
it's there?

Reply:
<img width="683" alt="image" src="https://github.com/user-attachments/assets/72f6b08f-74e1-44ec-8872-f7d5c3c4cc2f">

Reply:
ah i didnt see it lol